sources/advanced/Command.ts

```diff
@@ -340,7 +349,7 @@ export abstract class Command<Context extends BaseContext = BaseContext> {
             }
         }
 
-        const exitCode = await this.execute();
+        const exitCode = await this.execute().catch(error => this.catch(error));
```

Comment:
I'm not sure this is the right place - it won't catch validation errors, for instance.

Reply:
Yes that's true. It should be higher to also catch validation errors.